scripts/flags/flags.js

```diff
@@ -0,0 +1,214 @@
+    return value;
+  } else {
+    throw new Error(`Unexpected WWW Modern value ${value} for flag ${flag}`);
+  }
+}
+
+function getWWWClassicFlagValue(flag) {
+  const value = ReactFeatureFlagsWWW[flag];
+  if (value === true) {
+    return '✅';
+  } else if (value === false || value === 'experimental') {
+    return '❌';
+  } else if (value === 'profile') {
+    return '⏱️';
+  } else if (value === 'dev') {
+    return '💻';
+  } else if (value === 'gk') {
+    return '🧪';
+  } else if (typeof value === 'number') {
+    return value;
+  } else {
+    throw new Error(`Unexpected WWW Classic value ${value} for flag ${flag}`);
+  }
+}
+
+function getRNOSSFlagValue(flag) {
+  const value = ReactFeatureFlagsNativeOSS[flag];
+  if (value === true) {
+    return '✅';
+  } else if (value === false || value === 'experimental') {
+    return '❌';
+  } else if (value === 'profile') {
+    return '⏱️';
+  } else if (value === 'dev') {
+    return '💻';
+  } else if (value === 'gk') {
+    return '🧪';
+  } else if (typeof value === 'number') {
+    return value;
+  } else {
+    throw new Error(`Unexpected RN OSS value ${value} for flag ${flag}`);
+  }
+}
+
+function getRNFBFlagValue(flag) {
+  const value = ReactFeatureFlagsNativeFB[flag];
+  if (value === true) {
+    return '✅';
+  } else if (value === false || value === 'experimental') {
+    return '❌';
+  } else if (value === 'profile') {
+    return '⏱️';
+  } else if (value === 'dev') {
+    return '💻';
+  } else if (value === 'gk') {
+    return '🧪';
+  } else if (typeof value === 'number') {
+    return value;
+  } else {
+    throw new Error(`Unexpected RN FB value ${value} for flag ${flag}`);
+  }
+}
+
+// iterate over allFlagUniqueFlags
+const table = [];
+// eslint-disable-next-line no-for-of-loops/no-for-of-loops
+for (const flag of allFlagsUniqueFlags) {
+  const row = {
+    flag,
+    'WWW Classic': getWWWClassicFlagValue(flag),
+    'RN FB': getRNFBFlagValue(flag),
+    'OSS Canary': getOSSCanaryFlagValue(flag),
+    'OSS Experimental': getOSSExperimentalFlagValue(flag),
+    'WWW Modern': getWWWModernFlagValue(flag),
+    'RN OSS': getRNOSSFlagValue(flag),
+  };
+  table.push(row);
+}
+
+const sortBy = 'WWW Classic';
+
+// sort table by OSS Experimental
+const sorted = table.sort((rowA, rowB) => {
+  return rowB[sortBy].toString().localeCompare(rowA[sortBy]);
+});
+
+if (argv.csv) {
+  const fs = require('fs');
+  const csvHeader =
+    'Flag name, WWW Classic, RN FB, OSS Canary, OSS Experimental, WWW Modern, RN OSS\n';
+  const csvRows = sorted.map(row => {
+    return `${row.flag}, ${row['WWW Classic']}, ${row['RN FB']}, ${row['OSS Canary']}, ${row['OSS Experimental']}, ${row['WWW Modern']}, ${row['RN OSS']}`;
+  });
+  fs.writeFile('./flags.csv', csvHeader + csvRows.join('\n'), function (err) {
+    if (err) {
+      return console.log(err);
+    }
+    console.log('The file was saved!');
+  });
+}
+
+// print table with formatting
+console.table(sorted);
```

Comment:
Minor style tip: if you pass an object to a table, it's using the keys for the first row.

Reply:
Updated, much better, and it doesn't have the `""`s around the key now, thanks!